events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js

```diff
@@ -530,17 +531,23 @@
             });
             return count;
         } else if (state === "begin") {
+            console.log("creating validators");
```

Comment:
are these logs needed? it's ok if the answer is yes

Reply:
It is useful in context that if anything goes wrong on game day it'll leave us a trial of breadcrumbs...